pkg/apis/componentconfig/cloudcore/v1alpha1/default.go

```diff
@@ -200,6 +209,12 @@ func NewMinCloudCoreConfig() *CloudCoreConfig {
 					Address: "0.0.0.0",
 				},
 			},
+			Router: &Router{
+				Enable:      true,
```

Comment:
Please set it to false for the first release.

Reply:
done